scripts/WinInstallFiles/InstallJMRI.nsi

```diff
@@ -50,6 +50,9 @@
 ; -------------------------------------------------------------------------
 ; - Version History
 ; -------------------------------------------------------------------------
+; - Version 0.1.25.0
```

Comment:
Need to also update the version number at https://github.com/JMRI/JMRI/pull/7432/files#diff-735059f12e8148114622f09cfca2fea4R310 to match, otherwise the built installer does not reflect this